src/main/resources/templates/gardens/list.html

```diff
@@ -12,10 +12,6 @@
 <div class="container body-content">
 
     <h2 th:text="${title}">Gardens</h2>
-
-<!--remove link later when admin is set up-->
-    <a th:href="@{'/gardens/add/'}">Add Garden</a>
```

Comment:
This line still needs to be in place since the Water Gardens link on the navbar leads to the list (this page). There's no other link to add a garden right now.